CourseGrab/Models/User.swift

```diff
@@ -62,31 +62,14 @@ struct User {
 // MARK: - Methods
 
 extension User {
-    
-    func initializeSession() {
-        guard let token = googleToken else {
-            print("No token")
-            return
-        }
-        NetworkManager.shared.initializeSession(googleToken: token).observe { response in
-            switch response {
-            case .value(let value):
-                if value.success {
-                    User.current?.sessionAuthorization = value.data
-                } else {
-                    print("No success")
-                }
-            case .error(let error):
-                print(error)
-            }
-        }
-    }
 
     func signOut() {
+        NetworkManager.shared.enableNotifications(enabled: false).observe { result in
+            // should we ensure that notifications are disabled before signing out?
```

Comment:
The only reason to make sure that notifications are disabled is so that the user doesn't get notifications for sections that are linked to an old account. They wouldn't be able to turn off those notifications without signing back in with the old account first, which could get confusing.